src/components/shared/Logout.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+  AlertDialog,
+  AlertDialogAction,
+  AlertDialogCancel,
+  AlertDialogContent,
+  AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  AlertDialogTrigger,
+} from "@/components/ui/alert-dialog";
+import { Button } from "../ui/button";
+
+export function LogOut({ fnc }: { fnc: () => void }) {
+  return (
+    <AlertDialog>
+      <AlertDialogTrigger asChild>
+        <Button variant="ghost" className="shad-button_ghost">
+          <img src="/assets/icons/logout.svg" alt="logout" />
+          <p className="small-medium lg-base-medium"> Logout </p>
+        </Button>
+      </AlertDialogTrigger>
+      <AlertDialogContent>
+        <AlertDialogHeader>
+          <AlertDialogTitle>Are you sure to want to Logout?</AlertDialogTitle>
```

Comment:
The `AlertDialogTitle` contains a grammatical error. It should be “Are you sure you want to log out?” instead of “Are you sure to want to Logout?”

Reply:
@avayyyyyyy  fix request changes